SpchtDescriptorFormat.py

```diff
@@ -85,10 +85,7 @@ def __str__(self):
             return "SPCHT{_empty_}"
 
     def __bool__(self):
-        if self._DESCRI is not None:
-            return True
-        else:
-            return False
+        return self._DESCRI is not None
```

Comment:
this way is way more clear, this is slightly embarrassing for me as i wrote this part only a short while ago, i should have known better